matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/pushrules/RuleIds.kt

```diff
@@ -35,6 +35,8 @@ object RuleIds {
     // Default Content Rules
     const val RULE_ID_CONTAIN_USER_NAME = ".m.rule.contains_user_name"
 
+    const val RULE_ID_KEYWORDS = "_keywords"
```

Comment:
This is clearly not the same value format than the other RULE_ID. Is it really a rule Id?

Reply:
Sorry should have commented this. It's not really a rule Id but a placeholder just used client side. I should have commented this. Web has the same approach/naming.